src/code_80005FD0.c

```diff
@@ -561,22 +564,25 @@ s16 get_angle_between_waypoints(Vec3f arg0, Vec3f arg1) {
     return phi_v1;
 }
 
-s32 func_80006018(f32 arg0, f32 arg1, f32 arg2, f32 arg3, f32 arg4, f32 arg5, f32 arg6, f32 arg7) {
-    f32 temp_f0;
+bool is_collide_with_vehicle(f32 vehicleX, f32 vehicleY, f32 vehicleVelocityX, f32 vehicleVelocityZ, f32 distanceX,
```

Comment:
This is likely x1 y1, and x2 y2.

Velocity does not make sense here. This is likely a circle to circle collision check.

is_colliding_with_vehicle

Reply:
Velocity it's what we give in input so I just re take the same name